VCFv4.3.tex

```diff
@@ -547,6 +606,7 @@ \section{INFO keys used for structural variants}
 
 \footnotesize
 \begin{verbatim}
+##INFO=<ID=TDLOC,Number=1,Type=String,Description="Tandem duplication type (must be: direct, inverted-upstream, or inverted-downstream)">
 ##INFO=<ID=MEINFO,Number=4,Type=String,Description="Mobile element info of the form NAME,START,END,POLARITY">
```

Comment:
We should define the values and meanings of these. The specs don't define what POLARITY is.